src/java/org/apache/cassandra/schema/TableParams.java

```diff
@@ -140,7 +140,6 @@ public Builder unbuild()
 
     public void validate()
     {
-        compaction.validate();
```

Comment:
Please add a comment why we don't validate the compaction parameters (they are validated during `CompactionParams` construction).

Reply:
Fixed in a3502c76945a910700dfd9d53ae1e88bdeea8d03.